desktop/src/com/frostwire/gui/updates/UpdateManager.java

```diff
@@ -210,16 +212,16 @@ private void handlePossibleUpdateMessage(UpdateMessageReader umr, boolean force)
 
             // Logic for Windows or Mac Update
             if (OSUtils.isWindows() || OSUtils.isMacOSX()) {
-                if (hasUrl && !hasTorrent && !hasInstallerUrl) {
+                if ((hasUrl && !hasTorrent && !hasInstallerUrl) || hasUrl && !canUseTorrent()) {
```

Comment:
Well, we would not be entirely true with the option then. Maybe the user doesn't want any BT traffic. Even "safe" stuff. Also making the engine work just for that might be tricky. Would require a lot of reading of the code. (I don't know if running the `SessionManager` leaves any artifacts (JNI part))